AdobeStockImageAdminUi/Test/Unit/Ui/Component/Listing/Filter/ColorTest.php

```diff
@@ -177,7 +177,7 @@ public function testPrepare(?string $colorPickerMode, string $appliedValue): voi
         $color->prepare();
     }
 
-    private function verifyApplyFilter(string $appliedValue, Filter $filter, ContextInterface $context): void
+    private function verifyApplyFilter(string $appliedValue, Filter $filter, MockObject $context): void
```

Comment:
```suggestion
    private function verifyApplyFilter(string $appliedValue, Filter $filter, ContextInterface $context): void
```
Almost every object in the test is `MockObject`, `ContextInterface` is more specific type

Reply:
It's not 100% correct. ContextInterface is not inheriting MockObject and vice versa. 
So you have to choose one. Actually "MockObject" methods are used inside this method, that's why it's more correct to use MockObject here.
`ContextInterface` doesn't have `expects` method